src/Form/Input.js

```diff
@@ -65,6 +85,38 @@ const AutogrowShadow = createComponent({
   }),
 });
 
+const LeftIcon = createComponent({
+  name: 'LeftIcon',
+  as: Icon,
+  style: ({ onClick }) => css`
+    position: absolute;
+    left: 8px;
+    top: 50%;
+    transform: translateY(-50%);
+
+    ${onClick &&
+      css`
+        cursor: pointer;
+      `}
+  `,
+});
+
+const RightIcon = createComponent({
+  name: 'RightIcon',
+  as: Icon,
+  style: ({ onClick }) => css`
+    position: absolute;
+    right: 8px;
+    top: 50%;
+    transform: translateY(-50%);
+
+    ${onClick &&
+      css`
+        cursor: pointer;
+      `}
```

Comment:
Styles for both left and right icons are __almost__ identical. Can we consolidate somehow?

Reply:
Yeah, let me give it a shot 👍